library/spdm_responder_lib/libspdm_rsp_set_certificate.c

```diff
@@ -151,6 +152,13 @@ libspdm_return_t libspdm_get_response_set_certificate(libspdm_context_t *spdm_co
     root_cert_hash_size = libspdm_get_hash_size(
         spdm_context->connection_info.algorithm.base_hash_algo);
 
+    if ((libspdm_get_connection_version(spdm_context) >= SPDM_MESSAGE_VERSION_13) &&
+        (spdm_context->connection_info.multi_key_conn_rsp)) {
+        key_pair_id = spdm_request->header.param2;
+        /*write KeyPairID.*/
+        spdm_context->local_context.local_key_pair_id[slot_id] = key_pair_id;
+    }
```

Comment:
I think we need to check version >= 1.3 && multi_key_conn_rsp == false.

Reply:
Thanks. I have added the check.